src/main/java/com/cglx/Calculator.java

```diff
@@ -37,5 +37,23 @@ public int square(int a) {
 		return result;
 	}
 
+	public int sqroot(int a) {
+
+		return (int) Math.sqrt(a);
+	}
+
+	private void readFile() throws IOException {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused private "readFile" method. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-01:9990/coding_rules#rule_key=squid%3AUnusedPrivateMethod)